app/plugins/okta-custom-provider.js

```diff
@@ -0,0 +1,42 @@
+// this is work done by Tim Costa (https://github.com/timcosta)
+// in a PR for Bell https://github.com/hapijs/bell/pull/450 to support Okta custom authorisation servers
+// this should be the default Okta provider in the next major release but has not been merged yet
+// in the meantime manually add the provider
+const Joi = require('@hapi/joi')
+
+const internals = {
+  schema: Joi.object({
+    uri: Joi.string().uri().required(),
+    authorizationServerId: Joi.string().alphanum()
+  }).required()
+}
+
+exports = module.exports = function (options) {
```

Comment:
Not sure why the okta custom provider uses this form for module exports. I can see it's a multiple assign but I don't know what it achieves here.